packages/wrangler/src/config/config-helpers.ts

```diff
@@ -23,9 +30,9 @@ export function resolveWranglerConfigPath(
 		script?: string;
 	},
 	options: { useRedirect?: boolean }
-): string | undefined {
+): ConfigPaths {
 	if (config !== undefined) {
-		return config;
+		return { userConfigPath: config, configPath: config };
 	}
```

Comment:
Does this mean that config redirection won't happen if the user specifies `--config` directly? 

That seems reasonable, just wanted to double check

Reply:
Yes that is precisely the behaviour.